pyvcloud/vcd/org.py

```diff
@@ -582,6 +610,39 @@ def get_roles_query(self, name_filter=None):
             qfilter=org_filter)
         return query, resource_type
 
+    def get_right(self, right_name):
+        """
+        Retrieves corresponding record of the specified right.
+        :param right_name: (str): The name of the right record to be retrieved
+        :return: (dict): Right record in dict format
+        """  # NOQA
+        right_record = self.list_rights(('name', right_name))
+        if len(right_record) < 1:
+            raise Exception('Right \'%s\' does not exist.' % right_name)
+        return right_record[0]
+
+    def list_rights(self, name_filter=None):
+        """
+        Get the typed query for the rights in the current Org
+        :param name_filter: (tuple): (name ,'right name') Filter the rights by
```

Comment:
I don't think we need a tuple here, rather a support for multiple filters (by name) which can be logically or-ed will be more valuable. 

fyi : Rights are unique in context of an org, so this list will always contain 1 or 0.